tests/app/submitter/test_converter.py

```diff
@@ -131,6 +131,18 @@ def test_display_address_should_be_set_in_payload_metadata(
     ]
 
 
+def test_instrument_id_should_be_set_in_payload_collection_if_form_type_in_metadata(
```

Comment:
We should have one for not existing. We have a test for set and not set for `started_at`.